packages/engine/Source/Core/Cartesian3.js

```diff
@@ -892,7 +892,9 @@ Cartesian3.fromDegrees = function (
 
 let scratchN = new Cartesian3();
 let scratchK = new Cartesian3();
-const wgs84RadiiSquared = new Cartesian3(
+
+// To prevent a circular dependency, this value is overridden by Ellipsoid when Ellipsoid.default is set
+Cartesian3._ellipsoidRadiiSquared = new Cartesian3(
```

Comment:
Would it make more sense to have break the dependency by having a `DefaultEllipsoidValues` file that both `Ellipsoid` and `Cartesian` depend on?

Putting an ellipsoid property on the Cartesian class made these classes feel to closely coupled

Reply:
We would still run into a similar problem because, to avoid the workaround, that new file would need to depend on `Ellipsoid`.

Reply:
We could omit the values, but the circular dependency is still functionally there. 

The only way to remove it entirely would be to break the public API by moving `fromDegrees` or `fromRadians` out of that class. Theoretically better archectecture, but breaking the public API which I'm erring on the side of avoiding since those function have been in place for years.